src/main/java/org/spongepowered/common/service/pagination/ActivePagination.java

```diff
@@ -160,6 +160,8 @@ protected Text calculateFooter(int currentPage) {
         TextBuilder ret = Texts.builder();
         if (hasPrevious) {
             ret.append(this.prevPageText).append(DIVIDER_TEXT);
+        } else {
+            ret.append(Texts.of("«")).append(DIVIDER_TEXT);
```

Comment:
I think these are supposed to be stored in a field set in the constructor (see line 71).